roles/runtime/tasks/main.yaml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Register as user with password and auto-subscribe to available content.
+  redhat_subscription:
+    state: present
+    username: <username>
+    password: <password>
```

Comment:
Please address the following things:
- Read the username and password from a group_vars/all file
- This block should get executed only if the running machine is rhel
- Before this block add a check: throw an error message if username or password is not set if the distro is a rhel